stdlib/2/exceptions.pyi

```diff
@@ -1,80 +1,49 @@
-from typing import Any, Tuple, Optional
+from __builtin__ import ArithmeticError
```

Comment:
This needs to be `ArithmeticError as ArithmeticError` so that the name is re-exported from this module.

Reply:
Ah, that's the test failure was about.  I'll send another pull request.

Reply:
https://github.com/python/typeshed/pull/1611